.github/workflows/integration-tests.yml

```diff
@@ -75,9 +85,7 @@ jobs:
 
       - name: Run integration tests (single site)
         if: ${{ matrix.php != 8.0 }}
-        run: composer test
-      - name: Run integration tests (single site with code coverage)
+        run: composer testwp
+      - name: Run integration tests (multisite site with code coverage)
```

Comment:
Would it be possible to test for multi *and* single site?

Reply:
It would, but right now, there's only one test that runs on multisite and skipped on single, so it's really not worth running 99.5% of the same tests twice.